internal/flow/notify.go

```diff
@@ -32,3 +33,77 @@ func (s *Service) NotifyCommitter(ctx context.Context, event *http.PodNotifyRequ
 
 	return nil
 }
+
+func (s *Service) NotifyFluxEvent(ctx context.Context, event *http.FluxNotifyRequest) error {
+	span, ctx := s.Tracer.FromCtx(ctx, "flow.NotifyFluxEvent")
+	defer span.Finish()
+
+	// If there's no commits, let's just skips
+	if len(event.Commits) == 0 {
+		return errors.Errorf("no commits found")
+	}
+
+	// Range commit
+	for _, commit := range event.Commits {
+		commitMessage, err := parseCommitMessage(commit.Message)
+		if err != nil {
+			return err
+		}
+		email := commitMessage.GitAuthor
+		if !strings.Contains(email, "@lunar.app") {
+			//check UserMappings
+			lwEmail, ok := s.UserMappings[email]
+			if !ok {
+				log.WithContext(ctx).Errorf("%s is not a Lunar email and no mapping exist", email)
+				return errors.Errorf("%s is not a Lunar email and no mapping exist", email)
+			}
+			email = lwEmail
+		}
```

Comment:
The Slack client handles user mappings when getting the slack id, so this can be omitted.